freeze.py

```diff
@@ -93,6 +93,7 @@
                    "zmq",
                    "webbrowser",
                    "json",
+                   "packaging"
```

Comment:
Nit: trailing comma, please.
```suggestion
                   "packaging",
```

Reply:
Thanks for the suggestion. Resolved.